src/toil/common.py

```diff
@@ -410,7 +410,8 @@ def addOptions(parser: ArgumentParser, config: Config = Config()):
         description="Allows the specification of the minimum and maximum number of nodes in an autoscaled cluster, "
                     "as well as parameters to control the level of provisioning."
     )
-    provisioner_choices = ['aws', 'gce']
+    provisioner_choices = ['aws', 'gce', None]
```

Comment:
Do you want to document this version in the CLI options docs? Or the other one like is there now?

Reply:
I'll add in some more description.